### PR TITLE
Added nf-test for untarfiles

### PR DESCRIPTION
**Pull request by GallVp**
Added nf-test for untarfiles with #6226

### CODE REVIEW

File: modules/nf-core/untarfiles/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
+    test("test-untarfiles-different-output-path") {
```
Comment (famosab) [inline modules/nf-core/untarfiles/tests/main.nf.test:34]: I think I dont get the naming of the test and the idea behind the test. Does this not do the same thing as test-untarfiles?

Comment (GallVp) [reply modules/nf-core/untarfiles/tests/main.nf.test:34]: Thank you @famosab 

I think the idea is to test the case where the tar has multiple subdirectories in it.

Comment (famosab) [reply modules/nf-core/untarfiles/tests/main.nf.test:34]: ok maybe the test can be renamed then to: 
```suggestion
    test("test-untarfiles-subdirectories") {
```

**Comment by maxulysse** [comment]
I was more thinking about getting rid of this modules, untar can work it out just as nicely

**Comment by GallVp** [comment]
> I was more thinking about getting rid of this modules, untar can work it out just as nicely

Good idea. Should we add a deprecation notice and merge?

File: modules/nf-core/untarfiles/tests/main.nf.test (hunk 1 above)
Comment (GallVp) [reply modules/nf-core/untarfiles/tests/main.nf.test]: Good idea. That's done now.

**Comment by SPPearce** [comment]
@maxulysse , @GallVp , are we adding a deprecation notice here?

**Comment by GallVp** [comment]
> @maxulysse , @GallVp , are we adding a deprecation notice here?

Done Thank you!

**Review by SPPearce — APPROVED** [review]

**Pull request merged**